recipe/build.sh

```diff
@@ -27,7 +27,7 @@ cmake \
     -DPIO_ENABLE_TESTS:BOOL=OFF \
     -DPIO_ENABLE_EXAMPLES:BOOL=OFF \
     -DPIO_ENABLE_TIMING:BOOL=OFF \
-    -DWITH_PNETCDF:BOOL=OFF \
+    -DWITH_PNETCDF:BOOL=ON \
```

Comment:
@zklaus, should this be conditional on `mpi != mpi_serial`?

Reply:
That is:
```
if [[ ! -z "$mpi" && "$mpi" == "mpi_serial" ]]; then
  export USE_MPI_SERIAL="ON"
  export WITH_PNETCDF="OFF"
else
  export USE_MPI_SERIAL="OFF"
  export WITH_PNETCDF="ON"
fi
```
...
```
-DWITH_PNETCDF:BOOL=${WITH_PNETCDF}
```

Reply:
Good catch!